.github/workflows/pre-release-workflow.yml

```diff
@@ -31,9 +34,11 @@ jobs:
           echo $mavenSigningKeyRingFileEncoded | base64 -di > "$RUNNER_TEMP"/keystore/2DE631C1.gpg
           echo "mavenSigningKeyRingFile=$RUNNER_TEMP/keystore/2DE631C1.gpg" >> $GITHUB_ENV
 
-      - name: Checkout Branch
+      - name: Create and Checkout Branch
         uses: actions/checkout@v3
         with:
+          run: | 
+            git checkout -b release/${{ github.event.inputs.version_to_release }}
```

Comment:
@cesarmax22 this is where I'm creating the release branch and at that point, everything should be running on that and not modifying master.

Reply:
I need to change this so the branch name doesn't have the patch version - will update in a bit after we talk this through.